frontend/src/modules/userManagement/views/addUserView.js

```diff
@@ -19,17 +19,36 @@ define(function(require){
     },
 
     isValid: function() {
-      var email = this.$('input[name=email]').val().trim();
-      var valid = Helpers.isValidEmail(email);
-      if(valid) {
+      var valid = true;
+      var validFirstName = this.$('input[name=firstName]').val().trim().length > 0;
+      var validLastName = this.$('input[name=lastName]').val().trim().length > 0;
+      var validEmail = Helpers.isValidEmail(this.$('input[name=email]').val().trim());
+      var validPassword = this.$('input[name=password]').val().trim().length > 0;
+
+      this.$('.field-error').addClass('display-none');
+
+      if (!validFirstName) {
+        this.$('#firstNameError').removeClass('display-none');
+        valid = false;
+      }
+
+      if (!validLastName) {
+        this.$('#lastNameError').removeClass('display-none');
+        valid = false;
+      }
+
+      if (!validEmail) {
+        this.$('#emailError').removeClass('display-none');
+        valid = false;
+      }
+
+      if (!validPassword) {
+        this.$('#passwordError').removeClass('display-none');
+        valid = false;
+      }
```

Comment:
Would you mind tweaking this to be extensible? Something iterative like the below:
```js
this.$('.field-error').each(function(index, element) {
  var $error = $(element);
  var $input = $error.siblings('input');

  var isValid = $input.attr('name') === 'email' ?
    Helpers.isValidEmail($input.val().trim()) :
    $input.val().trim().length > 0;

  $error.toggleClass('display-none', isValid);

  if (!isValid) {
    valid = false;
  }
});
```

We can then lose the IDs in the DOM.